includes/manage_pages.inc

```diff
@@ -59,23 +59,23 @@ function islandora_paged_content_manage_pages_menu(AbstractObject $object) {
         '#collapsed' => TRUE,
       ),
       'page_progression' => array(
-        '#access' => user_access(ISLANDORA_METADATA_EDIT),
+        '#access' => islandora_datastream_access(ISLANDORA_METADATA_EDIT, $object['RELS-EXT']),
```

Comment:
Should this include a `isset($object['RELS-EXT'])` like you did [above](https://github.com/Islandora/islandora_paged_content/pull/139/files#diff-2d1af91ff8774fef646ae080ceae487bR50)? I'm not super sure but thought I would mention it

Reply:
I'm just realizing I think I merged your PR to 7.x and it was probably exactly like this. So I'll leave this up to you if you think it should be changed.

Reply:
mmm islandora_datastream_access() does an is_object() at https://github.com/Islandora/islandora/blob/9b37dcc9b853e109ce5ba3e1ae889703803ca3a5/islandora.module#L1716.
Seems like that when calling a function with an argument that is an array (or accessor here), PHP only evaluates it when its actually used? Not even sure if I did this right but a snippet: https://gist.github.com/DiegoPino/424aa47d559a308080ee11fc06cfc451  that piece does not fail even if RELS-EXT does not exist at all But i did in a hurry and tested in my TextMate PHP processor, so who knows. If an isset() is actually needed, then the parent 7.x will need that too.

Reply:
Yeah, had a [bit of discussion about this issue on one of the other PRs](https://github.com/Islandora/islandora_fits/pull/67#discussion_r144022803): No, it's not necessary, due to the internals of `islandora_datastream_access()`.